assets/css/climatehub.css

```diff
@@ -1,9 +1,44 @@
-/* Example Section */
-.someCoolClassName {
-  color: black;
-  font-family: Helvetica;
+.navbar-brand.shrink {
+  max-height: 75px;
+  transition: all 0.4s ease;
+  -webkit-transition: all 0.4s ease;
 }
 
-#someId {
-  width: 100%;
+.navbar-nav > li {
+  align-items: flex-end;
+  display: flex;
+  margin: 0 5px;
+  position: relative;
+}
+
+.navbar-light .navbar-nav li a::before {
+  border-top: 5px solid #000;
+  bottom: 125px;
+  box-shadow: 5px 0 0 #000 inset, -5px 0 0 #000 inset,  0 104px 0 #fff inset, 0 114px 0 #fff inset;
+  content: "";
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: bottom 0.5s ease-in-out;
+  z-index: -1;

```

Comment:
You can remove `.navbar-light`. When writing hex colours, please capitalize how characters, so it would be `#FFF`. The position, bottom, right, left, and top should all be grouped together for cleanliness.